src/drug_discovery/abfe.py

```diff
@@ -343,26 +348,10 @@ def parse_progress(cls, job: Job) -> dict:
     def _name_job(cls, job: Job) -> str:
         """utility function to name a job using inputs to that job"""
         try:
-            return f"ABFE run using <code>{job._metadata[0]['protein_id']}</code> and <code>{job._metadata[0]['ligand1_id']}</code>"
+            return f"ABFE run using <code>{job._metadata[0]['protein_file']}</code> and <code>{job._metadata[0]['ligand_file']}</code>"
         except Exception:
             return "ABFE run"
 
-    def _connect(self):
-        """connect to datahub and fetch Job IDs for this protein and ligand"""
-
-        # fetch from ABFE first
-        df = pd.DataFrame(
-            api.get_dataframe(
-                "ABFE",
-                return_type="dict",
-            )
-        )
-        df = df[df["Protein"] == self.parent.protein._do_id]
-        ligand_ids = [ligand._do_id for ligand in self.parent.ligands]
-        df = df[df["Ligand1"].isin(ligand_ids)]
-        job_ids = df[utils.COL_JOBID].tolist()
-        self._make_jobs_from_ids(job_ids)
-
```

Comment:
the connect method is gone, too, because we're not talking to databases on datahub